net/prosody/patches/010-remove-default-cflags.patch

```diff
@@ -0,0 +1,11 @@
+--- a/configure
++++ b/configure
+@@ -23,7 +23,7 @@ EXCERTS="yes"
+ PRNG=
+ PRNGLIBS=
+ 
+-CFLAGS="-fPIC -Wall -pedantic -std=c99"
++CFLAGS?="-fPIC -Wall -pedantic -std=c99"
+ LDFLAGS="-shared"
```

Comment:
Does we need not LDFLAGS+="-shared"  on this line as well for` TARGET_LDFLAGS += -L$(STAGING_DIR)/usr/lib`?
I have not compiled it so it might work.

Reply:
I believe TARGET_LDFLAGS gets added after that one. I'll check later.